tests/internal/test_native.py

```diff
@@ -1,122 +1,147 @@
-import os
+import pytest
 
-from ddtrace.internal.native import _apply_configuration_from_disk
-from ddtrace.internal.native import get_configuration_from_disk
 
-
-def test_get_configuration_from_disk__priority(tmp_path):
+@pytest.mark.subprocess(env={"DD_VERSION": "b"})
+def test_get_configuration_from_disk_managed_stable_config_priority():
     """
     Verify the order:
     local stable config < environment variables < managed stable config
     """
+    import os
+    import tempfile
 
-    local_config = tmp_path / "local_config.yaml"
-    local_config.write_text(
-        """
+    # Create managed config
+    with tempfile.NamedTemporaryFile(suffix=".yaml", prefix="managed_config") as managed_config:
+        managed_config.write(
+            b"""
 apm_configuration_default:
-  DD_SERVICE: "a"
```

Comment:
Currently DD_SERVICE is overriden in subprocess tests. To workaround this limitation I updated all tests to use `DD_VERSION` instead.